src/client/src/apps/MainRoute/widgets/blotter/components/blotterUtils.ts

```diff
@@ -1,22 +1,18 @@
 import { ColDef, CsvExportParams, ProcessCellForExportParams } from 'ag-grid-community'
-import { DateTime } from 'luxon'
+import { utcFormat } from 'd3-time-format'
 import { Trade, TradeStatus } from 'rt-types'
 import SetFilter from './filters/SetFilter'
 import numeral from 'numeral'
 import { capitalize } from 'lodash'
 
-const dateRenderer = (value: Date) => {
-  return DateTime.fromJSDate(value).toFormat('dd-MMM hh:mm:ss')
+function UtcFormatDate(date: Date, format: string = '%b %e, %H:%M:%S') {
+  return utcFormat(format)(date)
```

Comment:
I only just changed this to use luxon in #1200, is there a reason to use d3-time-fornat instead?

If there is then you should add it to the project deps because I removed it

Reply:
@BenGladman I put back the luxon date. Thanks!